config/database.yml

```diff
@@ -5,21 +5,24 @@
 #   gem "sqlite3"
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
+  username: postgres
```

Comment:
Do not push username password changes in database.yml

Reply:
where to give credentials

Reply:
@Smokashi23 either credentials.yml.enc file or .env file